.gitignore

```diff
@@ -10,6 +10,7 @@ __pycache__/
 *.so
 
 # Distribution / packaging
+orbax/
```

Comment:
Let's land this feature in Orbax first and then land it in MaxText. I'm totally agree in principle in MaxText for now.

Reply:
Sounds good, thanks.